server/lib/pdf.js

```diff
@@ -55,6 +56,11 @@ export const getConsolidatedInvoicesData = async fromCollective => {
     if (!HostCollectiveId) {
       continue;
     }
+
+    if (moment(transaction.createdAt).isSame(new Date(), 'month')) {
+      continue;
+    }
+
```

Comment:
This one also seems to be something we can easily check with SQL: 
`createdAt: { [Op.lt]: moment().startOf('month') }`